Editor/PostInstallWizard.cs

```diff
@@ -0,0 +1,190 @@
+            
+            Debug.Log(PlayerSettings.Android.minSdkVersion.ToString());
+            
+            GUIHyperlinkClick();
+
+            m_tickImage = AssetDatabase.LoadAssetAtPath<Texture2D>( tickPath );
+            m_crossImage = AssetDatabase.LoadAssetAtPath<Texture2D>( crossPath );
+        }
+
+#if UNITY_2020_1_OR_NEWER
+        void GUIHyperlinkClick()
+        {
+            EditorGUI.hyperLinkClicked += OnHyperLinkClicked;
+        }
+
+        void OnDestroy()
+        {
+            EditorGUI.hyperLinkClicked -= OnHyperLinkClicked;
+        }
+
+        private void OnHyperLinkClicked(EditorWindow window, HyperLinkClickedEventArgs args)
+        {
+            if(window!=this){
+                return;
+            }
+            Application.OpenURL( args.hyperLinkData["href"] );
+        }
+#else
+        void GUIHyperlinkClick()
+        {
+            //not availble before 2020.1
+        }
+#endif
+
+        public static void RefreshFiles(){
+            assetsAndroidManifestPath = Path.Combine(ProjectPath(),"Plugins/Android/AndroidManifest.xml");
+            
+            hasExistingManifest = File.Exists(assetsAndroidManifestPath);
+            if(hasExistingManifest){
+                existingManifestContents = File.ReadAllText(assetsAndroidManifestPath);
+            }
+            else {
+                Debug.LogWarning($"File not found {assetsAndroidManifestPath}");
+            }
+
+        }
+
+        public void OnGUI()
+        {
+            var height = EditorGUIUtility.singleLineHeight;
+            GUIStyle style = new GUIStyle( GUI.skin.label ) { richText = true };
+            GUIStyle bold = new GUIStyle(style){ fontStyle = FontStyle.Bold };
+                    
+            if( hasExistingManifest ){
+                EditorGUILayout.LabelField("AndroidManifest.xml file found",bold);
+                EditorGUILayout.LabelField("Manifest Bluetooth permissions:");
+                foreach(var androidPermission in m_androidPermission){
+                    if((int)PlayerSettings.Android.targetSdkVersion<androidPermission.minSdkVersion){
+                        continue;
+                    }
+                    bool hasPerm = androidPermission.Exists(existingManifestContents);
+                    EditorGUILayout.GetControlRect(GUILayout.Height(4));             
+                    EditorGUILayout.BeginHorizontal();
+                    //var rect = EditorGUILayout.GetControlRect(GUILayout.Width(height));
+                    androidPermission.foldOutOpen = EditorGUILayout.Foldout(androidPermission.foldOutOpen, $"{androidPermission.description}");
+                    //EditorGUILayout.LabelField($"{androidPermission.description} ",style);
+                    var rect = EditorGUILayout.GetControlRect(GUILayout.Width(height-4));
+                    rect.height = height-4;
+                    rect.y+=2;
+                    GUI.DrawTexture(rect,hasPerm ? m_tickImage : m_crossImage);
```

Comment:
This sounds like a much better idea than using a external texture. I'll implement immediately.

Reply:
done.

However it wasn't as straightforward as you might think, because Unity in their wisdom keep messing around with the availbile icon names in different Unity versions. I've settled on two for the tick. But as I was on 2021.3 the ones you suggested weren't available.

So we will have to keep an eye on this in future Unity releases, as there's no way to know if they are changing the name of an icon. It seems they have in the past.

Reply:
Good point, I didn't check for cross-version compatibility. Hopefully people will create an issue in case something pops up on newer versions. For now, this looks good and I'll approve / merge it!